mirrord/cli/src/main.rs

```diff
@@ -404,6 +405,43 @@ async fn register_to_waitlist(email: EmailAddress) -> Result<()> {
     Ok(())
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+struct VerifiedConfig {
+    config: Option<String>,
+    errors: Option<String>,
+}
+
+/// Verifies a config file specified by `path`.
+///
+/// ## Usage
+///
+/// ```sh
+/// mirrord verify-config ./config.json
+/// ```
+async fn verify_config(VerifyConfigArgs { path }: VerifyConfigArgs) -> Result<()> {
+    let mut config_context = ConfigContext::default();
+
+    let read_path = path.clone();
+    let verified = match async move { LayerFileConfig::from_path(path) }
```

Comment:
this seems to be a reverse functional programming which is functional but a mess to read..

Reply:
Yup, this was just pure chaos while I was thinking on how to keep the errors and read the file.

Nuked :boom: